exporters/otlp/otlpgrpc/options.go

```diff
@@ -60,41 +59,42 @@ const (
 }`
 )
 
-type grpcConnectionConfig struct {
+type config struct {
 	canDialInsecure    bool
 	collectorAddr      string
 	compressor         string
 	reconnectionPeriod time.Duration
-	grpcServiceConfig  string
-	grpcDialOptions    []grpc.DialOption
+	serviceConfig      string
+	dialOptions        []grpc.DialOption
 	headers            map[string]string
 	clientCredentials  credentials.TransportCredentials
 }
 
-type GRPCConnectionOption func(cfg *grpcConnectionConfig)
+// Option applies an option to the gRPC driver.
+type Option func(cfg *config)
```

Comment:
Should we take this opportunity to convert `Option` into an interface to bring it in line with our [style guide](https://github.com/open-telemetry/opentelemetry-go/blob/master/CONTRIBUTING.md#option)?

Reply:
Let it be done as a part of #536, please.